openedx/core/djangoapps/credit/partition_schemes.py

```diff
@@ -0,0 +1,151 @@
+"""
+Partition scheme for in-course reverification.
+
+This is responsible for placing users into one of two groups,
+ALLOW or DENY, for a partition associated with a particular
+in-course reverification checkpoint.
+
+NOTE: This really should be defined in the verify_student app,
+which owns the verification and reverification process.
+It isn't defined there now because (a) we need access to this in both Studio
+and the LMS, but verify_student is specific to the LMS, and
+(b) in-course reverification checkpoints currently have messaging that's
+specific to credit requirements.
+
+"""
+from django.core.cache import cache
+
+import logging
```

Comment:
Nice to have: https://github.com/edx/edx-platform/wiki/Python-Guidelines#imports-order